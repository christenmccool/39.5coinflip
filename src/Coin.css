.Coin-image {
  width: 150px;
}