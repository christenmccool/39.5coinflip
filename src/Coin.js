import './Coin.css';
import tailsImage from './tails.png';
import headsImage from './heads.png';

const Coin = (props) => {

  const image = props.side === "heads" ? headsImage : tailsImage;

  return (
    <div className="Coin">
      <img src={image} alt={`${props.side}`} className="Coin-image" />
    </div>
  )
}

export default Coin;