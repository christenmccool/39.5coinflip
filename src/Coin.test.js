import { render, fireEvent } from '@testing-library/react';
import Coin from './Coin';

test("renders without crashing", () => {
  render(<Coin />)
})

test("matches snapshot", () => {
  const {asFragment} = render(<Coin />);
  expect(asFragment()).toMatchSnapshot();
})