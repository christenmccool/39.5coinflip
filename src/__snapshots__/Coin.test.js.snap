// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Coin"
  >
    <img
      alt="undefined"
      class="Coin-image"
      src="tails.png"
    />
  </div>
</DocumentFragment>
`;
